src/iuse.cpp

```diff
@@ -4142,6 +4142,34 @@ int iuse::chop_tree( player *p, item *it, bool t, const tripoint &pos )
     return it->type->charges_to_use();
 }
 
+int iuse::chop_logs( player *p, item *it, bool t, const tripoint &pos )
+{
+    if( !p || t ) {
+        return 0;
+    }
+
+    tripoint dirp = pos;
+    if( !choose_adjacent( _( "Chop which which tree trunk?" ), dirp ) ) {
+        return 0;
+    }
+
+    int moves;
+
+    const ter_id ter = g->m.ter( dirp );
+    if( ter == t_trunk ) {
+        /** @EFFECT_STR reduces time required to chop down a tree */
+        moves = MINUTES( 70 - p->str_cur ) * 2 / it->get_quality( AXE ) * 100;
```

Comment:
I think this equation (used here and in `chop_tree`) could use some balance love.

Strength has a tiny effect (1 minute per cut for each strength).  Quality has a massive effect.

Reply:
Do you have something exact in mind?

Reply:
Maybe move strength to the denominator, something like:
```MINUTES( 70 ) / ( ( it->get_quality( AXE ) * 10) + p->str_cur );```